packages/schema-derive/src/generate_api.rs

```diff
@@ -15,23 +15,46 @@ pub fn write_api_impl(input: Options) -> Block {
         {
             #[cfg(target_arch = "wasm32")]
             compile_error!("can't compile schema generator for the `wasm32` arch\nhint: are you trying to compile a smart contract without specifying `--lib`?");
-            use ::std::env::current_dir;
+            use ::std::env;
             use ::std::fs::{create_dir_all, write};
 
             use ::cosmwasm_schema::{remove_schemas, Api, QueryResponses};
 
-            let mut out_dir = current_dir().unwrap();
+            if env::args().find(|arg| arg == "--help").is_some() {
+                println!("USAGE:");
+                println!("    cargo schema [OPTIONS]");
+                println!();
+                println!("FLAGS:");
+                println!("    --basic");
+                println!("            Generate pure JSON schema files rather than the \"unified\" format.");
+                println!("    --help");
+                println!("            Print this helpfile.");
+                return;
+            }
```

Comment:
Souldn't this code live somewhere in the CLI code (close to some `main` function)? `write_api_impl` should take the switch as an argument.

I guess the issue is that the contract developer owns the binaries. Maybe we should change that.

Reply:
I think it would be super cool if `cargo schema` was a binary you install with `cargo install`, but I get the feeling that wouldn't be so easy to do